### DISCUSSION TITLE
Linux client: the OOBE for volunteers

### DISCUSSION THREAD

**Discussion by RichardHaselgrove** [BOINC/boinc#4963] [General]
This is going to touch on two previous issues, and I hope provoke some thinking:

#3105: On Linux, the manager (user?) cannot start the core client as a service
#2993: Computing prefs 2.0

It arises from https://boinc.berkeley.edu/forum_thread.php?id=13284, which at first glance seems simple: how to free one (only) from two GPUs for daytime CAD use. Easy - we have a cc_config.xml option for this. Further issues arise, however.

1) This is Ubuntu 19.04, BOINC v7.14.2. cc_config.xml isn't in 'the data directory'. Find out where to put it (I told him to use the GUI).

2) cc_config.xml itself is in a read-only folder (/etc/boinc-client)  for ordinary users. However, it can be edited after creation through the sym-link in /var/lib/boinc-client

3) This particular option requires a client restart to activate.

We got through all that, but at the end of the day - how do you re-activate the reserved GPU? https://boinc.berkeley.edu/forum_thread.php?id=13285 - edit the read-only file again, restart the client again. It didn't take the first time - we suspect a finger-fumble stopping the client - but it did work eventually.

This particular use-case doesn't seem unreasonable: use one GPU while the machine is 'in use', use both GPUs when the machine is idle. It would also cover 'watch movie on one screen, check social media on the other'. Can we include it when we work up the cited issues, please?

**Comment by AenBleidd** [comment]
Since this is just an open question, I'm converting it to discussion.
Later it's possible to create a new issue when some action points appear.
